public/css/dashboard-application.css

```diff
@@ -1,386 +1,196 @@
-	top: 34px;
-
-	border-top: 5px solid #efa516;
-}
-
-/* How left from field */
-fieldset {
-	padding-left: 15%;
-}
-@media (max-width: 768px) {
-	fieldset {
-		padding-left: 3%;
-	}
-}
-@media (max-width: 576px) {
-	fieldset {
-		padding-left: 0;
-	}
-}
-.radio-inline {
-	margin-bottom: 2vh;
-	margin-right: 2%;
-}
-
-@media (max-width: 1200px) {
-	body {
-		background-position: 550px 40px;
-	}
-}
-@media (max-width: 650px) {
-	body {
-		background-position: 400px 40px;
-	}
-	.header h3 {
-		width: 76%;
-	}
-}
-
-/* FORM FIELDS */
-.short-answer {
-	margin: 0 0 5px;
-}
-
-textarea {
-	resize: none;
-}
-
-.whitespace {
-	color: transparent;
+	width: 0;
+	opacity: 0;
+	cursor: pointer;
 }
 
-.form-control {
-	height: 35px;
-	margin-bottom: 10px;
-	border-radius: 12px;
+/* Create a custom radio button */
+.checkmark {
+	position: absolute;
+	top: 0;
+	left: -20px;
+	height: 25px;
+	width: 25px;
+	background-color: #eee;
+	border-radius: 30%;
 }
 
-.sublabel {
-	text-transform: none;
+/* On mouse-over, add a grey background color */
+.container:hover input ~ .checkmark {
+	background-color: #ccc;
 }
 
-textarea::placeholder,
-input::placeholder {
-	text-transform: uppercase;
-	color: #b1b1b1 !important;
+/* When the radio button is checked, add a green background */
+.container input:checked ~ .checkmark {
+	background-color: var(--green);
 }
 
-@media (max-width: 828px) {
-	.whitespace {
-		display: none;
-	}
+/* Create the indicator (the dot/circle - hidden when not checked) */
+.checkmark:after {
+	content: "";
+	position: absolute;
+	display: none;
 }
 
-/* BUTTONS */
-.buttons {
-	text-align: center;
-	padding-bottom: 30px;
+/* Show the indicator (dot/circle) when checked */
+.container input:checked ~ .checkmark:after {
+	display: block;
 }
 
-.resume {
-	background-color: #95b9cd;
-	border-color: #95b9cd;
-	text-transform: uppercase;
+/* Style the indicator (dot/circle) */
+.container .checkmark:after {
+	top: 9px;
+	left: 8.5px;
+	width: 8px;
+	height: 8px;
+	border-radius: 50%;
+	background: var(--white);
 }
 
-.buttons button {
-	margin: 0 60px;
-	width: 364px;
-	height: 54px;
-	margin-top: 50px;
-	border: none;
-}
-.submit-application {
-	color: white;
-	font-family: "Roboto", sans-serif;
-	background-color: #50b5dd;
-	display: inline-block;
-	padding: 8px 12px;
-	cursor: pointer;
+.form select {
 	border-radius: 12px;
-	font-style: normal;
-	font-weight: bold;
-	font-size: 18px;
-	border: none;
+	background-color: white;
+	width: 100%;
+	margin-bottom: 20px;
 }
 
-.buttons button:hover {
-	transition: 0.3s;
-	background-color: #78caeb;
+input[type="file"] {
+	display: none;
 }
 
-.file-upload {
-	color: #ffffff;
-	background-color: #50b5dd;
-	display: inline-block;
+button {
+	background-color: var(--green);
+	color: var(--white);
+	display: flex;
 	padding: 8px 12px;
 	cursor: pointer;
 	border-radius: 12px;
-	font-style: normal;
-	font-weight: bold;
-	font-size: 18px;
 	border: none;
+	transition: 0.3s ease;
+	font-weight: 700;
 }
```

Comment:
Make this rule more specific. Just having it as `button` makes it apply to buttons in the navbar, messing up styles there. Add a class to all of the buttons you want to apply it to or otherwise mention classes of wrapping `div`s